src/services/secret_manager.py

```diff
@@ -16,39 +16,40 @@
 from google.cloud import secretmanager
 from google.api_core.exceptions import GoogleAPICallError
 import logging
+from typing import Any
 
 class SecretManager:
-    def __init__(self, project_id):
+    def __init__(self, project_id: str) -> None:
         """
         Initialize the SecretManager with a Google Cloud project ID.
         A SecretManagerServiceClient is created for interacting with Secret Manager.
         """
-        self.client = secretmanager.SecretManagerServiceClient()
-        self.project_id = project_id
+        self.client: secretmanager.SecretManagerServiceClient = secretmanager.SecretManagerServiceClient()
```

Comment:
I think the type declaration is not necessary here, the type is inferred from the right-hand side of the assignment statement.